src/dune_findlib/dune

```diff
@@ -1,3 +1,3 @@
 (library
  (name dune_findlib)
- (libraries stdune dune_lang dune_engine memo ocaml dune_meta_parser))
+ (libraries stdune dune_lang dune_engine memo ocaml dune_meta_parser unix))
```

Comment:
As discussed, we should disable the warning for this library because it doesn't use Unix.

Reply:
done